src/hooks/permissionedPools/PermissionedPositionManager.sol

```diff
@@ -36,17 +37,25 @@ contract PermissionedPositionManager is PositionManager {
         PERMISSIONED_HOOKS = _permissionedHooks;
     }
 
-    /// @dev Disables transfers of the ERC721 liquidity position tokens
-    function transferFrom(address, address, uint256) public pure override {
-        revert("Transfer disabled");
+    /// @dev Only allow admins of permissioned tokens to transfer positions that contain their tokens
+    function transferFrom(address from, address to, uint256 id) public override onlyIfPoolManagerLocked {
+        (PoolKey memory poolKey,) = getPoolAndPositionInfo(id);
+        address admin1 = _getOwner(poolKey.currency0);
+        address admin2 = _getOwner(poolKey.currency1);
+        if (msg.sender != admin1 && msg.sender != admin2) {
+            revert Unauthorized();
+        }
+        getApproved[id] = msg.sender;
```

Comment:
also why is this happening here?

Reply:
ah yeah I see theres no internal func - is there really a need for an event though? on the transfer, the approval gets wiped